oop-homework/src/Person.java

```diff
@@ -0,0 +1,21 @@
+public class Person {
+    public String firstname;
```

Comment:
why are the data members public?
you are braking encapsulation. If you need them to be public please specify the reason via JavaDoc.

Reply:
Am gresit...nu trebuiau sa fie public; o sa modific